doc/python/shapes.md

```diff
@@ -638,6 +638,34 @@ fig.update_layout(
 fig.show()
 ```
 
+#### Adding Shapes to Subplots
+
+```python
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+
+# Create Subplots
+fig = make_subplots(rows=2, cols=2, start_cell='bottom-left')
```

Comment:
So why do we need to change the `start_cell` here? I would need an explanation, so probably the tutorial too ;-)

Reply:
It looked good to me also without changing the `start_cell`, so I'd suggest to limit the complexity here unless of course I am missing something.